stack/tasks/bundle-demos.js

```diff
@@ -43,7 +43,8 @@ async function buildComponents() {
           .then(result => result.css),
       }),
       babel({
-        runtimeHelpers: true,
```

Comment:
By removing this completely we loose reuse of babel-helpers

Reply:
Here is an interesting read about how babel's cli and helper optimisation is working:
https://brunoscopelliti.com/a-simple-babel-optimization-i-recently-learned/

Reply:
unfortunately official docs are rather obscure:
https://www.npmjs.com/package/babel-plugin-external-helpers